### ISSUE TITLE
Views Support

### ISSUE THREAD

**Issue by mrconnerton** [travist/drupal.api.js#2]
Any plans on supporting the views endpoint?

**Comment by travist** [comment]
Yeah, definitely...  I am just now doing some major re-factors to make it PhoneGap friendly so views is definitely on my radar on this one...
